### ISSUE TITLE
3.0 Reports: More reporting on subscriptions & churn

### ISSUE THREAD

**Issue by zackkatz** [easydigitaldownloads/easy-digital-downloads#7081]
The new reports are a huge improvement over 2.x, but the biggest questions I have as a business is not being addressed. Those questions center around churn and customers.

**The biggest for me: identifying churn**

- What is the subscription renewal rate over time, as a percent. I would love to see a "renewal rate over time" chart. This would make me _very_ happy.
- Absolute license growth over time ( active + pending - expired ), with lines showing the number of licenses with each status
- Surface regular, valuable customers who are not renewing ("Most Valuable Customers With Licenses that Expired")
- Updating the "New Customers" chart to also show a line of "Inactive Customers" who have no active licenses

The calculations in the new Overview dashboard are _great_, and I would like to see all the numbers that don't have charts become charts:

- Average revenue per customer
- Customer growth (absolute, not just New Customers like in the Customers tab)
- Average order value

Related to #6592

**Comment by pippinsplugins** [comment]
@zackkatz anything dealing with license keys or subscriptions will be additional reports registered and provided by the relevant extensions.. They won't be in core since most of that data doesn't exist unless those plugins are activated.

On that note, I absolutely want to have reports for everything you mentioned built, but they will be built outside the scope of core.

**Comment by zackkatz** [comment]
@pippinsplugins Makes sense! Then the bottom half still applies (charting the numbers in the Overview dashboard widgets).

Are the relevant extensions going to launch at the same time as 3.0? 🤞

**Comment by pippinsplugins** [comment]
We're anticipating launching updates to the extensions right after 3.0 to ensure there's a bit of a window to help dust settle. We'll be working on those reports likely during the 3.0 beta phase or near it.
